portfolio/src/main/webapp/index.html

```diff
@@ -8,10 +8,29 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>James Washington's Portfolio</h1>
+      <ul id="navbar">
+          <li id="navlink"><a href="index.html">Home</a></li>
+          <li id="navlink"><a href="hobbies.html">Hobbies</a></li>
+          <li id="navlink"><a href="foods.html">Favorite Foods</a></li>
+      </ul>
+      <img src="images/ProfilePic.jpg" alt="My Profile Picture" width="300" height="350">
+      <table style="width:80%.">
+          <tr>
+              <td> This is my portfolio. It'll include a lot of things about me.
+                    I'm born and raised in Kansas City, Missouri (I'm really from
+         	        a small town called Grandview, but it's close enough!). I have
+         	        3 siblings, of which I'm the oldest son. I have over 40 cousins,
+         	        so family gatherings are always a ton of fun! Education is big in
+         	        my family, so college is always pushed. I'm a rising junior and I
+         	        attend Morehouse College. I study Computer Science, and I plan on
+         	        going into industry when I graduate. I interned at Google Summer 2019,
+         	        and I'm interning Summer 2020 as well. Outside of these I don't have very
+         	        many projects.</th>
```

Comment:
I think this is supposed to be a </td>. It should also be on its own line indented evenly with the opening <td>